test/VoucherHub.test.ts

```diff
@@ -32,23 +32,17 @@ const initVoucherHubBalance = 10n * voucherValue; // arbitrary value, but should
 describe('VoucherHub', function () {
     let iexecPoco: string;
     let iexecPocoInstance: IexecPocoMock;
-    let voucherHubWithVoucherManagerSigner: VoucherHub;
-    let voucherHubWithAssetEligibilityManagerSigner: VoucherHub;
+    let voucherHubWithMinterSigner: VoucherHub;
+    let voucherHubWithManagerSigner: VoucherHub;
```

Comment:
```suggestion
    let voucherHubAsMinter: VoucherHub;
    let voucherHubAsManager: VoucherHub;
```
and `voucherHubAsAnyone`

Reply:
Yes I prefer that too, did not want to change your version :)
I'll do it now

Reply:
See [Update wording to "VoucherHub as role"](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/29/commits/22185d18b3233d185dac9fccc07f3fac1f381384) !